### PR TITLE
fix SDAM code formatting to get syntax highlighting

### PR DESCRIPTION
**Pull request by nbbeeken**
Fixed our code formatting to get some highlighting and the SDAM subroutines are now entries in the table contents under `Actions`.

None of the usual checklist applies since this does not have functional changes.

### CODE REVIEW

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst [hunk 1]
```diff
@@ -987,240 +987,247 @@ TopologyType ReplicaSetWithPrimary
...
+```````
 
 updateUnknownWithStandalone
-  This subroutine is executed
-  with the ServerDescription from Standalone
-  when the TopologyType is Unknown::
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subroutine is executed with the ServerDescription from Standalone
+when the TopologyType is Unknown
```
Comment (ShaneHarvey) [inline source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst:996]: Here and for the other code blocks let's keep a single `:`, like:
```
... when the TopologyType is Unknown:

.. code-block:: python
```

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]
Nice stuff!

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst (hunk 1 above)
Comment (nbbeeken) [reply source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst]: Agreed, fixed!

**Review by ShaneHarvey — APPROVED** [review]
LGTM!

**Pull request merged**